installer/conf/omsagent.d/patch_management.conf

```diff
@@ -27,7 +27,7 @@
 
 <source>
     @type tail
-    tag oms.update_progress
+    tag oms.apt_update_progress
```

Comment:
i'd prefer you follow the naming convention for the tag: oms.[feature].[sub feature].